docs/template/public/less/_side-nav.less

```diff
@@ -0,0 +1,201 @@
+@import '_variables';
+
+.kss-sidebar-shadow{
+    background: fade(@kss-shadow, 80%);
+    height: 100%;
+    width: 100%;
+    display: none;
+    position: fixed;
+    z-index: 900;
+    top: 0;
+    cursor: pointer;
+
+    .toggled &{
+        display: block;
+    }
+
+    @media(min-width:769px) {
+        display: none!important;
+    }
+}
+
+.kss-ref{
+  display: none;
+}
+
+#kss-nav {
+    background: @dark-blue none repeat scroll 0 0;
```

Comment:
this background class looks odd.  We should probably just use 'background-color' here instead